### PR TITLE
[15.0][MIG] product_harmonized_system: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Superseed: https://github.com/OCA/intrastat-extrastat/pull/158

@Tecnativa TT36971

### CODE REVIEW

File: product_harmonized_system/__manifest__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Copyright 2018-2020 brain-tec AG (http://www.braintec-group.com)
-# Copyright 2011-2020 Akretion (http://www.akretion.com)
-# Copyright 2009-2020 Noviat (http://www.noviat.com)
+# Copyright 2018-2021 brain-tec AG (http://www.braintec-group.com)
```
Comment (pedrobaeza) [inline product_harmonized_system/__manifest__.py]: Don't change copyright years for someone that hasn't touched the files

File: product_harmonized_system/models/hs_code.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
-# Copyright 2011-2020 Akretion France (http://www.akretion.com)
-# Copyright 2009-2020 Noviat (http://www.noviat.com)
+# Copyright 2011-2021 Akretion France (http://www.akretion.com)
+# Copyright 2009-2021 Noviat (http://www.noviat.com)
```
Comment (pedrobaeza) [inline product_harmonized_system/models/hs_code.py]: The same

File: product_harmonized_system/models/product_category.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# Copyright 2011-2020 Akretion France (http://www.akretion.com)
-# Copyright 2009-2020 Noviat (http://www.noviat.com)
+# Copyright 2011-2021 Akretion France (http://www.akretion.com)
+# Copyright 2009-2021 Noviat (http://www.noviat.com)
```
Comment (pedrobaeza) [inline product_harmonized_system/models/product_category.py]: The same

File: product_harmonized_system/models/product_template.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
-# Copyright 2011-2020 Akretion (http://www.akretion.com)
-# Copyright 2009-2020 Noviat (http://www.noviat.com)
+# Copyright 2011-2021 Akretion (http://www.akretion.com)
+# Copyright 2009-2021 Noviat (http://www.noviat.com)
```
Comment (pedrobaeza) [inline product_harmonized_system/models/product_template.py]: The same

File: product_harmonized_system/views/hs_code.xml [hunk 5]
```diff
@@ -54,50 +54,48 @@
...
+                    >
+                        <div class="o_field_widget o_stat_info">
+                            <span class="o_stat_value">
+                                <field name="product_tmpl_count" />
+                            </span>
+                            <span class="o_stat_text">Products</span>
+                        </div>
+                    </button>
+                </div>
+                <group name="main">
```
Comment (pedrobaeza) [inline product_harmonized_system/views/hs_code.xml]: The `<sheet>` shouldn't be removed here.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Add also your copyright

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@alexis-via please review that is everything on place for merging it.

File: product_harmonized_system/models/hs_code.py [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+        required=True,
+        help="Code used for the national Import/Export declaration. "
+        "The national code starts with the 6 digits of the H.S. and often "
+        "has a few additional digits to extend the H.S. code.",
+    )
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        "res.company",
+        string="Company",
+        default=lambda self: self.env.company.id,
```
Comment (rousseldenis) [inline product_harmonized_system/models/hs_code.py]: Why changing this behavior as now a company would be set. That was done at purpose to share these records.

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: product_harmonized_system/models/hs_code.py (hunk 6 above)
Comment (pedrobaeza) [reply product_harmonized_system/models/hs_code.py]: I think that change was done by @alexis-via . Isn't it? Which is the reason, Alexis?

Comment (rousseldenis) [reply product_harmonized_system/models/hs_code.py]: Yes, but in preceding PR version you reintroduced a default lambda with self.env.company.id. Alexis suppressed the default at all.

In last version it's ok. Thanks

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**